use-timescale/ha-replicas/high-availability.md

```diff
@@ -13,52 +13,59 @@ cloud_ui:
 # Manage high availability
 
 For Timescale Cloud Service with very low tolerance for downtime, Timescale Cloud offers 
-High Availability(HA) replicas. HA replicas significantly reduce the risk of downtime and data loss due to 
+High Availability (HA) replicas. HA replicas significantly reduce the risk of downtime and data loss due to 
 system failure, and enable services to avoid downtime during routine maintenance.
 
 This page shows you how to choose the best high availability option for your Timescale Cloud Service.  
 
 ## What is HA replication?
 
-HA replicas are exact, up-to-date copies of your database that automatically take over operations as your primary data 
-node if the original primary data node becomes unavailable. HA replicas are synchronous or asynchronous hot standbys 
-hosted in multiple AWS availability zones(AZ) that use streaming replication to minimize the chance of data loss during 
-failover. That is, the primary node streams its write-ahead log (WAL) to the replicas.
+HA replicas are exact, up-to-date copies of your database hosted in multiple AWS availability zones (AZ) within the same region as your primary node. They automatically take over operations if the original primary data node becomes unavailable. The primary node streams its write-ahead log (WAL) to the replicas to minimize the chances of data loss during failover. 
+
+HA replicas can be synchronous and asynchronous. 
+
+- Synchronous replicas: the primary commits its next write once the replica confirms that the previous write is complete. There is no lag between the primary and the replica. They are in the same state at all times. This is preferable if you need the highest level of data integrity. However, this affects the primary ingestion time.
+  
+- Asynchronous replicas: the primary commits its next write without the confirmation of the previous write completion. The asynchronous HA replicas often have a lag, in both time and data, compared to the primary. This is preferable if you need the shortest primary ingest time.
+
+image from the doc draft
```

Comment:
Can you add the image please. Just give it an imperfect name so we can delete it if necessary.